interfaces/HO-Portal/src/app/program/program-people/program-people.component.ts

```diff
@@ -200,38 +328,63 @@ export class ProgramPeopleComponent implements OnChanges {
           pa: `PA #${index + 1}`,
           score: person.score,
           did: person.did,
-          created: formatDate(person.created, this.dateFormat, this.locale),
-          updated: formatDate(person.updated, this.dateFormat, this.locale),
+          processStarted: formatDate(person.created, this.dateFormat, this.locale),
+          digitalIdCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          digitalIdValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          inclusionCommunication: person.inclusionDate ? formatDate(person.inclusionDate, this.dateFormat, this.locale) : null,
+          included: person.included ? 'Included' : (person.excluded ? 'Excluded' : ''),
```

Comment:
These strings should be translatable, or event only be set in the template, not here. (preferably)